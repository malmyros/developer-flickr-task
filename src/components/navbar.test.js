import React from 'react';
import { shallow } from 'enzyme';
import Navbar from './navbar';

describe('navbar', () => {
  it('should render without crashing', () => {
    const component = shallow(<Navbar/>);
    expect(component).toMatchSnapshot();
  });
});
