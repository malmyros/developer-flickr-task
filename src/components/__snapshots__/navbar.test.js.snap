// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar should render without crashing 1`] = `
<div
  className="navbar navbar-dark bg-dark shadow-sm"
>
  <div
    className="container d-flex justify-content-between"
  >
    <a
      className="navbar-brand d-flex align-items-center"
      href="/"
      title="PhotoStreamer"
    >
      <strong>
        PhotoStreamer
      </strong>
    </a>
  </div>
</div>
`;
